client/blocks/reader-post-card/conversation-post.jsx

```diff
@@ -20,7 +20,16 @@ class ConversationPost extends React.Component {
 	};
 
 	render() {
-		const commentIdsToShow = map( take( this.props.comments, 3 ), 'ID' );
+		const commentIdsToShow = map(
+			take(
+				filter(
+					this.props.comments,
+					comment => comment.type !== 'trackback' && comment.type !== 'pingback'
+				),
+				3
```

Comment:
actually, i'm looking at the code, id rather not waste time on it. 
we are going to need to rip all of the "how many comments to show" logic out anyway and I have no idea what that'll end up looking like.

should it be a prop? will this component itself be given all the ids? will everything be based off postKey? etc etc.